tests/subsys/zbus/unittests/include/zbus_channels.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2022 Rodrigo Peixoto <rodrigopex@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+ZBUS_CHAN_DEFINE(version,	       /* Name */
+		 false,		       /* On changes only */
+		 true,		       /* Read only */
+		 struct version,       /* Message type */
+		 NULL,		       /* Validator */
+		 ZBUS_OBSERVERS_EMPTY, /* observers */
+		 ZBUS_MSG_INIT(.major = 0, .minor = 1,
+			       .build = 1023) /* Initial value major 0, minor 1, build 1023 */
+)
```

Comment:
I assume this shows that Zbus channels can only be defined in this single header. Have you thought about using iterable sections?